lightning/src/ln/onchaintx.rs

```diff
@@ -666,6 +666,7 @@ impl<ChanSigner: ChannelKeys> OnchainTxHandler<ChanSigner> {
 		let mut new_claims = Vec::new();
 		let mut aggregated_claim = HashMap::new();
 		let mut aggregated_soonest = ::std::u32::MAX;
+		let mut broadcasted = false;
```

Comment:
The more idiomatic past tense of broadcast is broadcast. However, I think a more easily parsable name would be `did_broadcast`.